manifests/Putty/Putty/0.73.yaml

```diff
@@ -0,0 +1,25 @@
+Id: putty.putty
+Version: 0.73
+Name: Putty
+Publisher: Plex
+License: MIT
+LicenseUrl: https://www.chiark.greenend.org.uk/~sgtatham/putty/licence.html
+AppMoniker: Putty
+Tags: Putty
+Description: PuTTY is a client program for the SSH, Telnet, Rlogin, and SUPDUP network protocols.
+Homepage: https://putty.org/
+Installers:
+  - Arch: x64
```

Comment:
winget don't support more than one installer at this time.[#706](https://github.com/microsoft/winget-pkgs/pull/706)
So at current moment the best way is adding x86 installer instead of x64 one for compatibility

Reply:
Thank you. 

I will make this change now.